docs/tutorial/part01.rst

```diff
@@ -2,5 +2,83 @@
 Set up the environment.
 =======================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+The following section will have the steps to set your local environment, so you can begin to work on your Django-neomodel implementation.
+If you want a point to start, we provide a base app to begin the tutorial. First clone the paradise-papers-django_ repo.
+
+    .. _paradise-papers-django: https://github.com/neo4j-examples/paradise-papers-django
+
+After you download the repo go to the ``start_app`` branch and you will get a working app that will be our start point.
+
+.. [*] Note:
+    you can use your own app to implement this tutorial and if you download the the start branch you will have only the base django project
+     with the front-end implementation.
+
+Set your virtualenv
+===================
+
+First you need Python and pip install on your local environment. Here is de official download page of python_

```

Comment:
I think it would sound better if it was " First, you need to install Python and pip in your local environment."  Also, put a comma after First. 

 Here is de official download page of python_ => change 'de' to 'the'

Reply:
Ups Spanglish 😅